loopin_test.go

```diff
@@ -326,11 +347,18 @@ func testLoopInResume(t *testing.T, state loopdb.SwapState, expired bool) {
 					},
 				},
 			},
-			Hash: testPreimage.Hash(),
+			Hash:            testPreimage.Hash(),
+			ProtocolVersion: storedVersion,
 		},
 	}
 
+	scriptVersion := swap.HtlcV1
```

Comment:
If we are setting the script version in the test, why don't we just pass in the htlc version? Since we're not actually testing our version -> htlc type mapping, just duplicating it in a test.

Reply:
done, ptal